### PR TITLE
SBERDOMA-443 add emergency_indicator, filter; fix text color for staus tag

### PR DESCRIPTION
**Pull request by leonid-d**
- add emergency tag
- add filter by emergency
- fix status tag text color
- add vertical-align for tickets table

### CODE REVIEW

File: apps/condo/domains/common/components/containers/GlobalStyle.tsx [hunk 1]
```diff
@@ -153,6 +153,10 @@ export default function GlobalStyle () {
                 }
               }
 
+              .ticket-table tbody.ant-table-tbody {
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/containers/GlobalStyle.tsx:156]: can we use local styles to table cell render?

File: apps/condo/domains/ticket/hooks/useEmergencySearch.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { useCallback, useState } from 'react'
+import { CheckboxChangeEvent } from 'antd/lib/checkbox/Checkbox'
+import { useRouter } from 'next/router'
+import qs from 'qs'
+import { pickBy, get, debounce } from 'lodash'
+import { getFiltersFromQuery } from '@condo/domains/common/utils/helpers'
+
+export const useEmergencySearch = <F>(loading): [boolean, (e: CheckboxChangeEvent) => void] => {
```
Comment (Dimitreee) [inline apps/condo/domains/ticket/hooks/useEmergencySearch.ts:8]: look good!

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 3]
```diff
@@ -126,13 +127,19 @@ export const useTableColumns = (sort: Array<string>, filters: IFilters) => {
...
                     return (
-                        <Tag color={backgroundColor} style={{ color }}>
-                            <Typography.Text strong>{status.name}</Typography.Text>
-                        </Tag>
+                        <span style={{ lineHeight: '30px' }}>
+                            <Tag color={backgroundColor}>
+                                <Typography.Text style={{ color }}>{status.name}</Typography.Text>
+                            </Tag>
+                            { record.isEmergency &&
+                            <Tag color={ '#FFCCC7' }>
```
Comment (Dimitreee) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:139]: use colors from styles consts

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 4]
```diff
@@ -126,13 +127,19 @@ export const useTableColumns = (sort: Array<string>, filters: IFilters) => {
...
+                                <Typography.Text style={{ color: '#F5222D' }}>{ EmergencyMessage }</Typography.Text>
```
Comment (Dimitreee) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:140]: use colors from styles consts

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 5]
```diff
@@ -126,13 +127,19 @@ export const useTableColumns = (sort: Array<string>, filters: IFilters) => {
...
                 filteredValue: getFilteredValue(filters, 'status'),
-                render: (status) => {
+                render: (status, record) => {
                     const { primary: color, secondary: backgroundColor } = status.colors
 
                     return (
-                        <Tag color={backgroundColor} style={{ color }}>
-                            <Typography.Text strong>{status.name}</Typography.Text>
-                        </Tag>
+                        <span style={{ lineHeight: '30px' }}>
```
Comment (Dimitreee) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:134]: use Space component from ant

File: apps/condo/pages/ticket/index.tsx [hunk 6]
```diff
@@ -170,10 +173,18 @@ const TicketsPage: IPageWithHeaderAction = () => {
                                             value={search}
                                         />
                                     </Col>
+                                    <Col span={1}></Col>
```
Comment (Dimitreee) [inline apps/condo/pages/ticket/index.tsx:176]: use offset={1} col props on next Col compoennt

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/common/constants/style.js [hunk 7]
```diff
@@ -24,6 +24,8 @@ const black = '#000'
 const beautifulBlue = '#eFF7FF'
 const markColor = '#B5CCFF'
 const whiteTranslucent = 'rgba(255, 255, 255, 0.9)'
+const emergencyTagBackground = '#FFCCC7'
```
Comment (pahaz) [inline apps/condo/domains/common/constants/style.js:27]: is it really a common domain?

**Review by pahaz — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

**Pull request merged**